core/include/grids/populator.hpp

```diff
@@ -8,10 +8,19 @@
 #pragma once
 
 #include "utils/indexing.hpp"
+#include "definitions/invalid_values.hpp"
+#include <vecmem/containers/device_vector.hpp>
+#include <vecmem/containers/jagged_device_vector.hpp>
+
+//#include <vecmem/containers/vector.hpp>
+//#include <vecmem/containers/jagged_vector.hpp>
+//#include <vecmem/containers/data/vector_buffer.hpp>
+//#include <vecmem/containers/data/jagged_vector_buffer.hpp>
```

Comment:
Let's not add the commented lines. :stuck_out_tongue:

Reply:
Sure

Reply:
Sure